### ISSUE TITLE
No woking

### ISSUE THREAD

**Issue by harukunz05asuna** [AlizerUncaged/desktop-waifu#64]
It says " No Python at 'C:\Users\flandre-real\AppData\Local\Programs\Python\Python310\python.exe' "
![saiugasiugu8gsa8cg](https://github.com/AlizerUncaged/desktop-waifu/assets/62566537/dfbaf921-993f-4267-a30d-885f6e4fb488)

**Comment by chancla2** [comment]
ok maybe i can help you, could you:
Steps:
1.- please open a cmd, write, python --version, you should see something like this, Python 3.11.3 (change if you have another version).
2.- type in cmd where python, it give something like thisC:\Users\Youruser\AppData\Local\Programs\Python\Python311\python.exe
3.-open "\desktop-waifu-master\waifu\pyvenv.cfg", it has 3 lines:
Home = "here paste step 2 path"
include-system-site-packages = false (dont move nothig)
version = "paste python --version"

at the end something like this:

home = C:\Users\Youruser\AppData\Local\Programs\Python\Python311
include-system-site-packages = false
version = 3.11.3

save and try again

**Comment by nobunagaa1** [comment]
i did the things you said but same thing not working

**Comment by nobunagaa1** [comment]
@chancla2 how do i change versions?

**Comment by chancla2** [comment]
did you install python on your pc?

**Comment by harukunz05asuna** [comment]
yes, now i have other issues, can i show you on discord what is the problem?
my discord is Forget#9045

**Comment by chancla2** [comment]
ok another guy has the same problem, look im new on the programing stuff,
but i can try to help this sunday with the other guy

**Comment by harukunz05asuna** [comment]
i did everything but in the end after api vtube studio when i press the key
it doesn't record.
